_raml/apps/products/products.raml

```diff
@@ -130,23 +130,51 @@ post:
     responses:
       204:
 
-  /crossselling:
-    displayName: Product crossselling
-    get:
-      is: [ locale, currency ]
-      queryParameters:
-        includeInvisible:
-          description: Includes invisible products. Only available with `products_write` authorisation.
-          type: boolean
-          required: false
-      description: Returns information on the crossselling of products. *epagesNow
-      securedBy: [products_read]
-      responses:
-        200:
-          body:
-            application/json:
-              example: !include examples/get_product_productid_crossselling_response.json
-              schema: !include schema/get_product_productid_crossselling.schema.json
+/crossselling:
+  displayName: Product crossselling
+  get:
+    is: [ locale, currency ]
+    queryParameters:
+      includeInvisible:
+        description: Includes invisible products. Only available with `products_write` authorisation.
+        type: boolean
+        required: false
+    description: Returns information on the crossselling of products. *epagesNow
+    securedBy: [products_read]
+    responses:
+      200:
+        body:
+          application/json:
+            example: !include examples/get_product_productid_crossselling_response.json
+            schema: !include schema/get_product_productid_crossselling.schema.json
+
+  put:
+    description: Replace all cross selling assignments
+    securedBy: [products_write]
+    body:
+      application/json:
+        example: !include examples/put_product_productid_crossselling_request.json
+        schema: !include schema/put_product_productid_crossselling_request_schema.json
+    responses:
+      200:
+        body:
+          application/json:
+            example: !include examples/get_product_productid_crossselling_response.json
+            schema: !include schema/get_product_productid_crossselling.schema.json
+
+  patch:
+    description: Add/Remove cross selling assignments
+    securedBy: [products_write]
+    body:
+      application/json:
+        example: !include examples/patch_product_productid_crossselling_request.json
+        schema: !include schema/patch_product_productid_crossselling_request_schema.json
+    responses:
+      200:
+        body:
+          application/json:
+            example: !include examples/get_product_productid_crossselling_response.json
+            schema: !include schema/get_product_productid_crossselling.schema.json
```

Comment:
```suggestion
            schema: !include schema/patch_product_productid_crossselling.schema.json
```
I think the same thing applies to the schema file.
**Note:** This schema file doesn't exist yet and needs to be created first!

Reply:
See above

Reply:
See comment for PATCH

Reply:
Updated